### ISSUE TITLE
Windows: upgrade to MSVC 2017

### ISSUE THREAD

**Issue by Be-ing** [mixxxdj/buildserver#47]
We are currently using MSVC 2015.

**Comment by Be-ing** [comment]
Shall we target this to 2.3? Any volunteers to take this up? @Pegasus-RPG?

**Comment by Be-ing** [comment]
Related: #59

**Comment by rryan** [comment]
Is anything blocked on this? We could, but I'd say it's pretty low priority given things are working fine with VS2015.

**Comment by Be-ing** [comment]
AFAIK nothing is currently blocked on this. But I'd like to be proactive and keep libraries and build tools up to date each release so we take care of them before they become problems.

**Comment by sblaisot** [comment]
I don't think you can have both VS2015 and VS2017 installed on the same system without some bad interaction problems.
So upgrading MSVS means either you create a new build slave server and target specifically this server for the builds needing the new MSVS (new major version) and keep the older branches on the older build server, or you upgrade the existing build slave and start building all branches and versions on the new MSVS, wich doesn't seems a good switch between minor releases.

**Comment by Be-ing** [comment]
I agree, switching the build tools for a minor release would not be great. I guess we'll need a new VM with the new compiler when we switch then.

**Comment by rryan** [comment]
mixxxdj/mixxx#1833 is the last step to fixing this.
